Vagrantfile

```diff
@@ -37,6 +37,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     shell.inline = <<-EOS
       set -e
       cd /vagrant/
+      # install/update all git-hooks
+      for hook in $(find git-hooks/ -type f | xargs basename)
```

Comment:
Have you run this and seen that it works? I came across this: https://stackoverflow.com/questions/37335560/how-to-extract-file-name-from-piped-results
which says xargs with basename can cause problems (I found  few other references to such problems, and suggestions to use other methods to get the filename itself).